candi/bashible/bashible.sh.tpl

```diff
@@ -41,7 +42,7 @@ function bb-event-error-create() {
           apiVersion: v1
           kind: Node
           name: '$(hostname -s)'
-          uid: "$(kubectl get node $(hostname -s) -o jsonpath='{.metadata.uid}')"
+          uid: "$(kubectl_exec get node $(hostname -s) -o jsonpath='{.metadata.uid}')"
```

Comment:
BTW, uid is not required and can be avoided.

Reply:
Nope. Without uid event is not binded to the object. I mean when you make `kubectl describe node X` - you will not fetch an event without id